askbot/jinja2/main_page/sidebar.html

```diff
@@ -1,9 +1,9 @@
 {% import "macros.html" as macros %}
 
 <div class="sidebar-content-block">
-    <h6 class="landing-message-header">How It Works</h6>
-    <p class="landing-message">Welcome to the Rover Q&amp;A Community! We’re building connections between dog lovers, and creating a space for support and camaraderie.</p>
-    <p class="landing-message">So feel free to take a look around, ask a question, or help others with your expertise. Our guidelines are simple: <strong>Be nice, be helpful, and celebrate our mutual love of dogs.</strong></p>
+    <h6 class="landing-message-header">Announcement</h6>
+
+    <p class="landing-message">The Rover Q&A Community is no longer accepting new posts or comments. Please see the <a href="https://support.rover.com">Rover Help Center</a> for product issues and <a href="/blog/">The Dog People</a> for fresh content.</p>
```

Comment:
For `/blog/` should we put in the full URL?

Reply:
I think is better if we leave it like /blog/ since this will redirect us to the right env for example if we are in staging

Reply:
Makes sense!